fg_lib/README.rst

```diff
@@ -0,0 +1,22 @@
+.. _fg_lib:
+
+Nordic Fuel Gauge library (FG lib)
+##################################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The fuel gauge library is the software complement of battery measurements made by a Nordic Power Management IC (PMIC).
+Measurements made during device operation are processed by this library, which results in a state-of-charge (SOC) prediction, along with other metrics.
+For the most accurate results, use a battery model that matches the battery used in the product.
+A battery model can be derived using the nPM Powerup App in `nRF Connect for Desktop`_ and `nPM1300 Evaluation Kit`_.
```

Comment:
is this "and" here inclusive or exclusive? Just checking to be 100% sure.

Reply:
You need both. nPM1300 Evaluation Kit is the hardware, controlled by nPM Powerup PC application.